nibabel/parrec.py

```diff
@@ -249,6 +249,18 @@ class PARRECError(Exception):
 GEN_RE = re.compile(r".\s+(.*?)\s*:\s*(.*)")
 
 
+def _unique_rows(a):
```

Comment:
Would be reassuring to have some tests of this function (I know it's exercised in the tests for the dimension labels, but still).